### PR TITLE
Set-Up-Associations Milestone 🗿 👨‍💻 

### PR DESCRIPTION
**Pull request by tsheporamantso**
## Set-Up-Associations Milestone 🗿 👨‍💻 

On this milestone 🗿, I have managed to implement the following:
- [ ] Ensure there are no linter errors 🏁 
- [ ] Followed correct Gitflow ✔️ 
- [ ] Adhered to Ruby best practices ✔️ 
- [ ] Add classroom file with class and method ➕ 
- [ ] Add relationship between classroom and student ➕ 🔗 
- [ ] Add Book file with class and method ➕ 
- [ ] Add Rental file with class and method ➕ 
- [ ] Add Associations between Book, Rental, Person  ➕ 🔗

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (Ghiftee) [inline book.rb]: - [x] Kindly set `attr_accessor` for rentals also as you would need to both access and modify it.

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```
Comment (Ghiftee) [inline rental.rb]: - [x] Kindly set `attr_accessor` for book and person too as you would need to both access and modify them.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @tsheporamantso 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ README is professional
✔️ Associations are created

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Ibizugbe — APPROVED** [review]
Hi @tsheporamantso 👋👋, 

approve2


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] attr_accessor has been set for `book` and `person` in `rental` class👍.
- [x] attr_accessor has been set for `rentals` in book class👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 1 above)
Comment (tsheporamantso) [reply book.rb]: Thank you will do.

File: rental.rb (hunk 2 above)
Comment (tsheporamantso) [reply rental.rb]: Thank you will do as requested

**Pull request merged**